### DISCUSSION TITLE
Why is unifuzz64.dll no longer needed.

### DISCUSSION THREAD

**Discussion by Github-Pat-T** [RichardOtter/Genealogy-scripts#9] [Q&A] [answered]
In comparing the 1.2.1 and 1.5.0 code, I don't see why unifuzz64.dll is no longer needed.
Does a new version of SQLite now handle the RMNOCASE collation?

**Comment by ricko2001** [comment, marked as answer]
Yes and no. 
The only relevant changes are in a couple of the SQL select commands. I just added a COLLATE NOCASE clause in the order by and it worked.
I'm getting the same results as before, but probably not using the rmnocase collated index.

**Comment by Github-Pat-T** [comment]
Thanks. 

⁣Get BlueMail for Android ​

**Comment by ve3meo** [comment]
I would say that UNIFUZZ as the substitute for the proprietary RMNOCASE collation is preferable to the NOCASE override. It comes closer to matching RMNOCASE. NOCASE is fine for the unaccented LATIN alphabet but does not support accents and other alphabets; RMNOCASE and UNIFUZZ do. Even having entered no characters with accents or from another alphabet, an empty RM database reindexed on NOCASE with SQLite fails the integrity test in RM itself until it is reindexed in RM. Iirc, that's only in the SourceTemplateTable index  but more errors will arise in other tables when different alphabets are used in data entry into fields that are index keys.

Failure to REINDEX at the start of a session that uses a collation that does not match that of the previous REINDEX can result in query errors and will cause the Integrity check (whether in a SQLite session or RM session) to fail.
